aptos-move/framework/aptos-framework/sources/zkid.move

```diff
@@ -0,0 +1,96 @@
+module aptos_framework::zkid {
+    use std::signer;
+    use aptos_framework::system_addresses;
+
+    #[resource_group(scope = global)]
+    struct ConfigGroup {}
+
+    #[resource_group_member(group = aptos_framework::zkid::ConfigGroup)]
+    /// The 288-byte Groth16 verification key (VK) for the zkID relation.
+    struct Groth16VerificationKey has key, store {
+        /// 32-byte serialization of `alpha * G`, where `G` is the generator of `G1`.
+        alpha_g1: vector<u8>,
+        /// 64-byte serialization of `alpha * H`, where `H` is the generator of `G2`.
+        beta_g2: vector<u8>,
+        /// 64-byte serialization of `gamma * H`, where `H` is the generator of `G2`.
+        gamma_g2: vector<u8>,
+        /// 64-byte serialization of `delta * H`, where `H` is the generator of `G2`.
+        delta_g2: vector<u8>,
+        /// 64-byte serialization of `\forall i \in {0, 1}, gamma^{-1} * (beta * a_i + alpha * b_i + c_i) * H`, where `H` is the generator of `G1`.
```

Comment:
\forall i \in {0, 1}, the 64-byte serialization of gamma^{-1} * (beta * a_i + alpha * b_i + c_i) * H